src/aiidalab_qe/guides/basic.html

```diff
@@ -0,0 +1,119 @@
+  "Confirm" to proceed to the next step.
+  <br />
+  <div class="alert alert-success">
+    <h4>Tasks</h4>
+    <ol>
+      <li>Click on the <b>From examples</b> tab</li>
+      <li>Select <b>Gold</b> from the dropdown list</li>
+      <li>Click the <b>Confirm</b> button to proceed.</li>
+    </ol>
+  </div>
+  <div class="alert alert-warning">
+    <b>Warning:</b> If the confirmed structure is not yet stored in the AiiDA
+    database, it will be stored automatically when you proceed to the next step.
+  </div>
+  <div class="alert alert-warning">
+    <b>Warning:</b> Changes after confirmation will unconfirm this step and
+    reset the following steps.
+  </div>
+</div>
+
+<div id="configuration-step">
+  In this step, we define the workflow tasks, including structure relaxation and
+  which properties to compute, and select the parameters of the calculations.
+  <div class="alert alert-success">
+    <h4>Tasks</h4>
+    <ol>
+      <li>Select <b>Full geometry</b> relaxation</li>
+      <li>Open <b>Step 2.1</b> to select properties</li>
+      <li>Open <b>Step 2.2</b> to customize parameters</li>
+      <li>Click <b>Confirm</b> to proceed</li>
+    </ol>
+  </div>
+  <div class="alert alert-warning">
+    <b>Note:</b> Changes after confirmation will unconfirm this step and reset
+    the following steps.
+  </div>
+</div>
+
+<div id="properties-selection">
+  Here we select the properties to calculate.
+  <div class="alert alert-success">
+    Select <b>Electronic band structure</b> and
+    <b>Projected density of states (PDOS)</b>
+  </div>
+</div>
+
+<div id="calculation-settings">
+  Here we can customize the calculation parameters.
+  <div class="alert alert-success">
+    Click on each tab to customize its settings.
+  </div>
+</div>
+
+<div id="basic-settings">
+  The basic settings panel provides top-level calculation settings including the
+  electronic and magnetic properties of the material. It also provides a choice
+  of three protocols that pre-configure many calculation settings, balancing
+  speed with accuracy.
+  <div class="alert alert-success">Select the <b>fast</b> protocol</div>
+  <div class="alert alert-warning">
+    Note: Due to the limited resources provided for the demo server, we select
+    the <b>fast</b> protocol to reduce the cost of the calculation.
+  </div>
+</div>
+
+<div id="advanced-settings">
+  The Advanced settings allow you to finely tune the calculation.
+  <div class="alert alert-warning">
+    In this walkthrough, we will not modify advanced settings and proceed with
+    the defaults.
+  </div>
+</div>
+
+<div id="bands-settings">
+  Here we configure the settings for computing the band structure.
+  <div class="alert alert-success">Check <b>Fat bands calculation</b></div>
+</div>
+
+<div id="pdos-settings">
+  Here we configure the settings for computing the projected density of states,
+  or PDOS.
+  <div class="alert alert-success">???</div>
```

Comment:
Similar to the advacned settings, 
```suggestion
  <div class="alert alert-success">In this walkthrough, we will not modify the pdos settings and proceed with the defaults.</div>
```